pokepaylib/src/main/java/jp/pokepay/pokepaylib/BankAPI/User/BankPayTopUp.java

```diff
@@ -28,6 +29,11 @@ public BankPayTopUp(@NonNull String id, @NonNull String accountId, @NonNull Stri
         this.amount = amount;
     }
 
+    public BankPayTopup RequestId(@NonNull String requestId) {
```

Comment:
Can change the method name to camelcase as per Java naming conventions?  `setRequestId` or `withRequestId` instead of `RequestId`?

Reply:
Oh, I have revert this pattern.
Which would be better?

Reply:
I think for now we leave it as it is.  I'll discuss with everyone later and decide whether we should update all APIs using builder.